utils/utils.css

```diff
@@ -32,12 +32,40 @@
 }
 .tuid__grid--dispaly {
   height: var(--g-height-xxxl);
-  width: var(--g-width-xxxl);
```

Comment:
g denotes global width and xxxl are very well explained and other variables have complete name for example if there is some value defined for buttons in specific then it is --button-width-xxxl in that way I have tried to incooprate most of the values. I am trying to perform a fallback mechanism so that every variable has customizable values if needed and they fall back to same predefined value if not explicitly defined by some user. 
Meanwhile I still need to port a lot of code in this format.
--g is just a overall coverage for the variables that are used genrally.